constellations/orion/src/domains/transaction.go

```diff
@@ -7,6 +7,14 @@ import (
 
 var TABLE_TRANSACTIONS = "transactions"
 
+const (
+	PAY_CASH   = "pay_cash"
+	PAY_CHECK  = "pay_check"
+	PAY_PAYPAL = "pay_paypal"
+	REFUND     = "refund"
+	CHARGE     = "charge"
+)
+
```

Comment:
Add the list of payments here (since it can then be used in multiple places of the codebase)

const ALL_TRANSACTION_TYPES = [...]